examples/events/main.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/vmware/govmomi/event"
+	"github.com/vmware/govmomi/examples"
+	"github.com/vmware/govmomi/vim25"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+// example use: go run main.go -url $GOVC_URL -p 8h -f VmEvent
+func main() {
+	begin := flag.Duration("p", time.Hour, "Begin time") // default BeginTime is 1h ago
```

Comment:
I had chosen `-p` as in `-previous $duration`. We could do `-begin` with a specific timestamp, but it looks like EndTime is ignored by `ReadNextEvents`.. and is meant only for [QueryEvents](https://vdc-download.vmware.com/vmwb-repository/dcr-public/b50dcbbf-051d-4204-a3e7-e1b618c1e384/538cf2ec-b34f-4bae-a332-3820ef9e7773/vim.event.EventManager.html#QueryEvent). Of course the vSphere doc doesn't mention that, but it is the behavior I'm seeing.

I'll go ahead and change `-p` to `-b` and otherwise leave the example as-is for now.

Reply:
SGTM